polyval/src/backend/soft64.rs

```diff
@@ -94,7 +104,7 @@ impl Drop for Polyval {
 
 /// 2 x `u64` values
 #[derive(Copy, Clone, Debug, Default, Eq, PartialEq)]
-struct U64x2(u64, u64);
+pub struct U64x2(u64, u64);
```

Comment:
This visibility change doesn't seem necessary?

Reply:
yeah, that was for debug :)